submissions/modding_inspiration/modding_inspiration.md

```diff
@@ -0,0 +1,7 @@
+## Modding Inspiration
```

Comment:
I think we could rework the title somewhat to be a bit more descriptive of what the piece actually is about. Exactly what to rework it too im not 100% sure but I'll think it over in case anything comes to me.